back/node_watcher/src/tasks/types.ts

```diff
@@ -123,13 +124,28 @@ export enum ProposalStatus {
   TABLED = 'Tabled',
 }
 
-export enum ReferendumStatus {
-  CANCELLED = ' Cancelled',
-  EXECUTED = 'Executed',
-  NOTPASSED = 'NotPassed',
-  PASSED = 'Passed',
-  STARTED = 'Started',
-}
+export const referendumStatus = {
+  CANCELLED: 'Cancelled',
+  EXECUTED: 'Executed',
+  NOTPASSED: 'NotPassed',
+  PASSED: 'Passed',
+  STARTED: 'Started',
+};
```

Comment:
That's not a type.. is that ok to have this here?
Also because it's not a type, I can't use it above for instance in the `NomidotReferendum`, I have to use a string. 
Is there is a nicer way to do?

Reply:
Yeah, this one is still alright i guess. What we also sometimes do, where there's shared logic, is put the shared logic in an `util/` folder, here `./util/referenda.ts`.

> I can't use it above for instance in the NomidotReferendum

You would need to use ReferendumStatus in NomidotReferendum.

The way you hardcoded `export type ReferendumStatus` is good. There is a pure typescript way ([here](https://www.typescriptlang.org/play/?ssl=10&ssc=1&pln=1&pc=1#code/KYDwDg9gTgLgBAYwgOwM7ysAZsTyAmArgLYDKMAhjIanALxwDeAUHG3AMICCAchwKIAZQfwAiALjgByDhWQJgAG0XB8UgDSt2-ABr8OAVQAqYyVP4hgCQjFUatbHgHkjABS6lSp6TwgxXFKiodprscO6e3lIBQSEOcKRGXABKJhLS5BSwcQC+cIGIKOgA3MzMMACeYMBwydi4wAQkmdS0DAAUldUQWHCYOHhEZJStAJQA2gDWwBU9cF3Ac-0NTcNUNAC6pUA)) but i'm not sure it's more readable, so it's okay to double hardcode them.

Reply:
Thanks, I've put all the statuses in a util file.